client/app/cdn/dedicated/manage/ssl/delete/cdn-dedicated-ssl-delete.controller.js

```diff
@@ -51,8 +59,10 @@ angular.module("App").controller("CdnDeleteSslCtrl", class CdnDeleteSslCtrl {
             serviceName: this.$stateParams.productId
         }).$promise.then((ssl) => {
             this.ssl = ssl;
+            this.actionEnabled = this.ssl.status === "on" || this.ssl.status === "off";
```

Comment:
What are the other states ? A Service would be useful to understand better how the SSL object works